test/verify/check-storage-btrfs

```diff
@@ -99,6 +99,30 @@ class TestStorageBtrfs(storagelib.StorageCase):
         b.go("#/")
         b.wait_visible(self.card("Storage"))
 
+        # podman subvolumes are hidden
+        m.execute(f"""
+        mkdir -p /var/lib/containers/storage/btrfs/subvolumes
+        btrfs subvolume create /var/lib/containers/storage/btrfs/subvolumes/containername
+
+        mkdir -p /home/admin/.local/share/containers/storage/btrfs/subvolumes
+        btrfs subvolume create /home/admin/.local/share/containers/storage/btrfs/subvolumes/grafana
+
+        btrfs subvolume create {subvol_mount_point}/pancake
+        """)
+
+        b.wait_in_text(self.card_row("Storage", name="pancake"), "btrfs subvolume")
+        b.wait_not_in_text("div[data-test-card-title='Storage']", "containername")
+        b.wait_not_in_text("div[data-test-card-title='Storage']", "grafana")
+
+        self.addCleanup(m.execute, """
```

Comment:
The cleanup should be registered before the `wait_in_text` statements so that it runs when those checks fail.

Reply:
The problem is that not all images are btrfs (ie. debian/ubuntu) so this has to be moved to /dev/sda which means we no longer need to cleanup \o/

But it also means we can't test /home/admin/.config but just match an alternative path

Reply:
> The problem is that not all images are btrfs

But that also means podman will not use btrfs for container storage there, right?